gwpy/cli/histogram.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+# vim: nu:ai:ts=4:sw=4
+
+#
+#  Copyright (C) 2021 Joseph Areeda <joseph.areeda@ligo.org>
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+"""The histogram CLI product
+"""
+
+
+__author__ = 'Joseph Areeda <joseph.areeda@ligo.org>'
+__email__ = 'joseph.areeda@ligo.org'
+__version__ = '0.0.1'
+__myname__ = 'histogram.py'
+
+import math
+import sys
+from scipy.stats import kurtosis, norm, skew
+
+from .cliproduct import TimeDomainProduct
+from ..plot import Plot
+from ..plot.tex import label_to_latex
+import warnings
+
+class Histogram(TimeDomainProduct):
+    """Plot a histogram of values for one or more time series
+    """
+    action = 'histogram'
+
+    def __init__(self, args):
+        super().__init__(args)
+        self.mean = None
+        self.std = None
+        self.skew = None
+        self.kurtosis = None
+        self.nbins = None
+        self.ymin = sys.maxsize
+        self.ymax = 0
+
+    @classmethod
+    def init_cli(self, parser):
+        """We add an option to add a fit ot gaussian PDF"""
+        super().init_cli(parser) # add all timeseries options
+        group = parser.add_argument_group('Curve fit options')
+        group.add_argument('--gauss', action='store_true',
+                           help='Add gaussian fit')
+
+    @classmethod
+    def get_xlabel(self):
+        """Text for x-axis label
+        """
+        units = self.units
+        if len(units) == 1 and str(units[0]) == '':  # dimensionless
+            return 'Sample value'
```

Comment:
It seems like lines 68-69 can be omitted because they fall into the use case on line 76, right?

```suggestion
```

Reply:
I don't think that's true. I rewrote that method to make it clearer. The situation is when a dimensionless channel has an emtpy string for units we want the label to be `Timeseries amplitude` without the separate test it would come out as `Timeseries amplitude[]`
I think it's better code now (not saying much)